packer.json

```diff
@@ -1,6 +1,6 @@
 {
     "builders": [{
         "type": "xenserver-iso",
-        "tools_iso_name": "guest-tools.iso"
+        "tools_iso_name": ""
```

Comment:
This file was kept from my initial fork. I believe it's unused since it isn't part of the hashicorp packer-plugin template. Can you remove this entirely?